packages/scaffold/src/client.ts

```diff
@@ -45,6 +51,24 @@ export class Web3ModalScaffold {
     ModalController.close()
   }
 
+  public getThemeMode() {
+    return ThemeController.state.themeMode
+  }
+
+  public getThemeVariables() {
+    return ThemeController.state.themeVariables
+  }
+
+  public setThemeMode(themeMode: ThemeMode) {
+    ThemeController.setThemeMode(themeMode)
+    setColorTheme(themeMode)
```

Comment:
Should these become part of the Controller methods, so they are always in sync? We could then remove them from everywhere else and just use ThemeController